app/views/reports/show.html.haml

```diff
@@ -1,10 +1,25 @@
-%h1 Settings for #{@report.name}
+%h1 Settings for Report »#{@report.name}«
+%hr
+%strong Individualize your Sensor-Live-Report! Change start date and variables.
+%hr
+%h3 General Settings
+%strong These settings determine the general look and function of your sensor-live-report.
 %p
   %strong Start Date:
   = @report.start_date
 %p
   %strong Video URL:
   = @report.video
+%p
+  =link_to edit_report_path(@report), {:class => "btn btn-primary btn-lg"} do
+    %i.fa.fa-cogs
+    Edit Settings
+%hr
+%h3 Markup Blocks
+%strong Markup Blocks (Text variables) are a great way to insert custom names in your text.components. Markup like key {exemplar} in a text-component will be replaced with its value "Bertha".
 = render 'variables'
-= link_to 'Edit', edit_report_path(@report), :class => "btn btn-primary"
+%p
+=link_to edit_report_path(@report), {:class => "btn btn-primary btn-lg"} do
+  %i.fa.fa-cogs
+  Edit Settings
```

Comment:
Ok, sorry for that. It feels just unclear how to edit the first part without a button. Whole settings page to be optimized ;)